subprojects/docs/src/docs/release/notes.md

```diff
@@ -2,6 +2,40 @@
 
 Here are the new features introduced in this Gradle release.
 
+### JUnit Platform and JUnit Jupiter Engine (a.k.a. JUnit 5) support
```

Comment:
This section should also give a hint on limitations.

Reply:
Added limitations on Java 8.